### DISCUSSION TITLE
2024-09-03-RevolutGuide2024

### DISCUSSION THREAD

**Discussion by giscus** [ouharu/ouharu.github.io#10] [Show and tell]
# 2024-09-03-RevolutGuide2024

声明：本文完成于2024年9月3日，如情况发生变化，请使用下方评论提醒更新。 引言 信息来源于小红书，介绍了日本到国内进行国际转账通过什么途径手续费最低以及如何保证对方收到的是日元而非转换为人民币（毕竟现在日元汇率低）。 目的是转账给国内亲属的账户获取转账证明后，满足报税时的亲属抚养扣除条件，可以减免在日所得税款，并且希望该过程的费用足够低，国内账户获取的日元取出后可以带回日本再次利用。 Revo

https://ouharu.github.io/2024-09-03-RevolutGuide2024

**Comment by ouharu** [comment]
补充：中国工商银行同样可以作为收款行
